boards/xtensa/intel_s1000_crb/doc/intel_s1000.rst

```diff
@@ -0,0 +1,216 @@
+   should pop-up on running this command. Otherwise, it means your system is missing some
+   package which is preventing successful installation. Most probably gtk2-i686 may be the
+   missing package. Please install it if it is not present.
+
+The xtensa processor inside S1000 is cavs21_LX6HiFi3_RF3_WB16. This processor doesn't come
+as part of the stock RF-2015.3 i.e. Xplorer-6.0.3 whose installation is described above.
+Follow the below steps to install cavs21_LX6HiFi3_RF3_WB16:
+
+1) Download the cadence XCC build for cavs21_LX6HiFi3_RF3_WB16 by contacting Intel.
+2) This will be a tgz file. Extract this to /opt/xtensa/XtDevTools/install/builds.
+3) Get inside /opt/xtensa/XtDevTools/install/builds/RF-2015.3-linux/cavs21_LX6HiFi3_RF3_WB16.
+4) Type sudo ./install. This is the Xtensa Processor Configuration Installation Tool
+   which is required to update the installation path. When it prompts to enter the path
+   to the Xtensa Tools directory, enter /opt/xtensa/XtDevTools/install/tools/RF-2015.3-linux/XtensaTools.
+   You should use the default registry /opt/xtensa/XtDevTools/install/tools/RF-2015.3-linux/XtensaTools/config.
+
+With the XCC toolchain installed, the Zephyr build system must be instructed
+to use this particular variant by setting the ``ZEPHYR_TOOLCHAIN_VARIANT``
+shell variable. Some more environment variables are also required (see below):
+
+.. code-block:: console
+
+   export XTENSA_PREFER_LICENSE=XTENSA
+   export ZEPHYR_TOOLCHAIN_VARIANT=xcc
+   export TOOLCHAIN_VER=RF-2015.3-linux
+   export XTENSA_CORE=cavs21_LX6HiFi3_RF3_WB16
+   export XTENSA_SYSTEM=/opt/xtensa/XtDevTools/install/tools/RF-2015.3-linux/XtensaTools/config/
+   export XTENSA_BUILD_PATHS=/opt/xtensa/XtDevTools/install/builds/
+   export XTENSA_OCD_PATH=/opt/Tensilica/xocd-12.0.4
+
+Flashing
+========
+
+The usual ``flash`` target will work with the ``intel_s1000_crb`` board
+configuration. Here is an example for the :ref:`hello_world`
+application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: intel_s1000_crb
+   :goals: flash
+
+Refer to :ref:`build_an_application` and :ref:`application_run` for
+more details.
+
+Setting up UART
+===============
+
+We use a "FT232RL FTDI USB To TTL Serial Converter Adapter Module" to
+tap the UART data. The J8 Header on S1000 CRB is dedicated for UART.
+Please do the connection between the J8 header and UART chip as shown below.
+
++------------+-----------+
+| UART chip  | J8 Header |
++============+===========+
+| DTR        |           |
++------------+-----------+
+| RX         | 2         |
++------------+-----------+
+| TX         | 4         |
++------------+-----------+
+| VCC        |           |
++------------+-----------+
+| CTS        |           |
++------------+-----------+
+| GND        | 10        |
++------------+-----------+
+
+We use minicom for monitoring the UART data. Follow the below steps
```

Comment:
Instead:

    Use ``minicom`` or another terminal emulator to monitor the UART data
    by following these steps: